demo/admin/src/products/future/generated/ManufacturersGrid.tsx

```diff
@@ -137,7 +138,27 @@ export function ManufacturersGrid(): React.ReactElement {
         },
         {
             field: "address_alternativeAddress_street",
-            headerName: intl.formatMessage({ id: "manufacturer.address.alternativeAddress.street", defaultMessage: "Alt-Street" }),
+            renderHeader: () => (
+                <>
+                    <GridColumnHeaderTitle
+                        label={intl.formatMessage({ id: "manufacturer.address.alternativeAddress.street", defaultMessage: "Alt-Street" })}
+                        columnWidth={150}
+                    >
+                        {intl.formatMessage({ id: "manufacturer.address.alternativeAddress.street", defaultMessage: "Alt-Street" })}
```

Comment:
You could now use `<FormattedMessage ... />` here, which is cleaner IMO

Reply:
FYI: Children of `GridColumnHeaderTitle` have been removed, as Ricky pointed out that they are not necessary.